.github/workflows/test.yml

```diff
@@ -11,13 +11,26 @@ on:
 
 env:
   GO_VERSION: 1.23.x
+  SHORT_TIMEOUT: 5
+  LONG_TIMEOUT: 60
 
 jobs:
-  lint:
-    runs-on: ubuntu-24.04
-    timeout-minutes: 20
+  lint-go:
+    timeout-minutes: "${{ fromJSON(env.SHORT_TIMEOUT) }}"
```

Comment:
Whoever came-up with this ridiculous notion that it would be cool to have the world CIs built on top of yaml + shell + a layer of crap deserves to be slapped repeatedly with a wet trout.